src/js/compat.ts

```diff
@@ -72,7 +76,67 @@ export async function initNodeModules() {
   };
 }
 
+/**
+ * If we're in Deno, we initialize the required modules.
+ * Otherwise, this does nothing.
+ * @private
+ */
+export async function initDenoModules() {
+  if (!IN_DENO) {
+    return;
+  }
+  // Workaround for https://github.com/emscripten-core/emscripten/issues/6241
+
+  let XHRModule = await import(
+    // @ts-ignore
+    /* webpackIgnore: true */ "https://deno.land/x/xmlhttprequest_deno@v0.0.2/mod.js"
+  );
+  (globalThis as any).XMLHttpRequest = XHRModule.default;
+
+  let denoStdlib = "https://deno.land/std@0.170.0/";
```

Comment:
@josephrocca just curious, in your experience, is it OK to hardcode the stdlib version used this way in Deno, or is there a better packaging solution?